src/@lekoarts/gatsby-theme-minimal-blog/components/post.tsx

```diff
@@ -1,99 +1,117 @@
-      }
-    }
-  }
-}
+        gatsbyImageData: any;
+      };
+    };
+    ns: string;
+  };
+};
 
+const px = [`32px`, `16px`, `8px`, `4px`];
+const shadow = px.map((v) => `rgba(0, 0, 0, 0.15) 0px ${v} ${v} 0px`);
 
-
-const px = [`32px`, `16px`, `8px`, `4px`]
-const shadow = px.map((v) => `rgba(0, 0, 0, 0.15) 0px ${v} ${v} 0px`)
-
-const Post: React.FC<React.PropsWithChildren<PageProps<MBPostProps>>> = ({ data: { post }, children }) => {
-  //const imageData = post.image ? getImage(post.image) : null;
+const Post: React.FC<React.PropsWithChildren<PageProps<MBPostProps>>> = ({
+  data,
+  children,
+}) => {
+  const post = data?.post;
+  const ns = post?.ns;
+  const isTranslationEnabled = !!ns;
+  const { t } = useTranslation(ns);
 
   return (
     <Layout>
       <Heading as="h1" variant="styles.h1">
-        {post.title}
+        {isTranslationEnabled ? t('title') : post.title}
       </Heading>
-      <p sx={{ color: `secondary`, mt: 3, a: { color: `secondary` }, fontSize: [1, 1, 2] }}>
+      <p
+        sx={{
+          color: `secondary`,
+          mt: 3,
+          a: { color: `secondary` },
+          fontSize: [1, 1, 2],
+        }}
+      >
         <time>{post.date}</time>
         {post.tags && (
           <React.Fragment>
             {` — `}
-            <ItemTags tags={post.tags} />
+            <ItemTags ns={ns} tags={post.tags} />
           </React.Fragment>
         )}
         {post.timeToRead && ` — `}
-        {post.timeToRead && <span>{post.timeToRead} min read</span>}
+        {post.timeToRead && (
+          <span>
+            {post.timeToRead} {isTranslationEnabled ? t('min_read') : ''}
+          </span>
+        )}
       </p>
-    
+
       <section
-      sx={{
-        my: 5,
-        "img, .gatsby-image-wrapper": {
-          my: [4, 4, 5],
-          borderRadius: `4px`,
-          boxShadow: shadow.join(`, `),
-        },
-        "img, .gatsby-image-wrapper img": {
-          borderRadius: `4px`,
-        },
-        variant: `layout.content`,
-      }}
-    >
-      {children}
-    </section>
+        sx={{
+          my: 5,
+          'img, .gatsby-image-wrapper': {
+            my: [4, 4, 5],
+            borderRadius: `4px`,
+            boxShadow: shadow.join(`, `),
+          },
+          'img, .gatsby-image-wrapper img': {
+            borderRadius: `4px`,
+          },
+          variant: `layout.content`,
+        }}
+      >
+        {children}
+      </section>
 
       <PostFooter post={post} />
     </Layout>
-  )
-}
-
-export default Post
+  );
+};
 
+export default Post;
 
 export const Head: HeadFC<MBPostProps> = ({ data: { post } }) => {
-  const firstTag = post.tags && post.tags[0]
-  const isBookReview = firstTag && firstTag.name === "Обзор книги"
+  const firstTag = post.tags && post.tags[0];
+  const isBookReview = firstTag && firstTag.name === 'Обзор книги';
 
-  const title = isBookReview ? `Краткое содержание - ${post.title}` : post.title
+  const title = isBookReview
```

Comment:
Post metadata needs to be translated as well.
![Screenshot by Dropbox Capture](https://github.com/romanr111/romanrud-blog/assets/12322543/85ba1934-7be5-4fce-afd5-503120ad5289)
![Screenshot by Dropbox Capture](https://github.com/romanr111/romanrud-blog/assets/12322543/c03dbf17-2051-4d7c-8332-32f35ec435b4)